### PR TITLE
OLS-800: Configure and deploy Openshift Lightspeed

### PR DESCRIPTION
**Pull request by rh-tokeefe**
CP to the TP branch

Version(s): Tech Preview

Issue: https://issues.redhat.com/browse/OLS-800

Link to docs preview:
https://78588--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/configure/ols-configuring-openshift-lightspeed

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: configure/ols-configuring-openshift-lightspeed.adoc [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 toc::[]
 
-// retaining for formatting
-// include::modules/ols-installing-operator.adoc[leveloffset=+1]
\ No newline at end of file
+Once the {ols-long} Operator is installed, deploying the {ols-long} service involves two steps. First, you create a secret using your Large Language Model (LLM) provider credentials. Then, you create the `OLSConfig` custom resource (CR). The CR contains information that informs the Operator how to deploy the {ols-long} service.
+
+[NOTE]
+====
+The instructions assume that you are installing {{ols-long}} using the `kubeadmin` user account. If you are using a regular user account with `cluster-admin` privileges, read the section of the documentation that discusses RBAC.
```
Comment (JoaoFula) [inline configure/ols-configuring-openshift-lightspeed.adoc:13]: There's one too many {} around ols-long

**Comment by JoaoFula** [comment]
Just the one small nitpick I commented above.
LGTM otherwise

File: configure/ols-configuring-openshift-lightspeed.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply configure/ols-configuring-openshift-lightspeed.adoc]: done

**Review by gabriel-rh — COMMENTED** [review]
LGTM ...Just one mandatory fix :-)

File: modules/ols-deploying-lightspeed-service.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ols-deploying-lightspeed-service_{context}"]
+= Deploying the OpenShift Lightspeed service
+
+.Prerequisites
+
+* You have installed the OpenShit Lightspeed Operator.
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ols-deploying-lightspeed-service.adoc]: Indeed! That should have been an attribute. Must have missed that one. It's an attribute now.

**Comment by thoraxe** [comment]
LGTM

File: modules/ols-completing-lightspeed-installation-process.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+
+* You have access to the {ocp-short-name} CLI (oc).
+
+* You have installed the {ols-long} Operator.
+
+* You have create the credentials secret and the OLSConfig custom resource configuration file.
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ols-completing-lightspeed-installation-process.adoc]: done

**Review by gabriel-rh — APPROVED** [review]
LGTM

**Pull request merged**